src/main/java/io/rebloom/client/Client.java

```diff
@@ -336,6 +340,132 @@ public String topkIncrBy(String key, String item, long increment) {
          .getMultiBulkReply();
    }
  }
+ 
+	//
+	// Count-Min-Sketch Implementation
+	//
+
+	@Override
+	public void cmsInitByDim(String key, Long width, Long depth) {
+		try (Jedis conn = _conn()) {
+			String rep = sendCommand(conn, CMSCommand.INITBYDIM, //
+					SafeEncoder.encode(key), //
+					Protocol.toByteArray(width), //
+					Protocol.toByteArray(depth)).getStatusCodeReply();
+
+			if (!rep.equals("OK")) {
+				throw new JedisException(rep);
+			}
+		}
+	}
+
+	@Override
+	public void cmsInitByProb(String key, Double error, Double probability) {
+		try (Jedis conn = _conn()) {
+			String rep = sendCommand(conn, CMSCommand.INITBYPROB, //
+					SafeEncoder.encode(key), //
+					Protocol.toByteArray(error), //
+					Protocol.toByteArray(probability)).getStatusCodeReply();
+
+			if (!rep.equals("OK")) {
+				throw new JedisException(rep);
+			}
```

Comment:
Make return type `boolean`.
```suggestion
      return rep.equals("OK");
```

Reply:
> Make return type `boolean`.

I also wanted to do that but all prior implementations of commands that return OK or an error were handled this way, so I just tried to keep it consistent.